.github/workflows/snapshots.yml

```diff
@@ -25,12 +27,12 @@ jobs:
         with:
           go-version: stable
           check-latest: true
-      - run: scripts/build_test_images.sh
```

Comment:
this is run by `scripts/run_tests` when `TEST_ACCEPTANCE` is `true`, so we can omit it here